recipes/cuik-molmaker/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cuik_molmaker" %}
+{% set version = "0.2" %}
+{% set rdkit_version = rdkit %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/cuik-molmaker/archive/refs/tags/v0.2.tar.gz
+  sha256: c137442f06af5f195efe7a916aefae065c8c52b40ae8e028a5bcd672b75c5fe8
+
+build:
+  script_env:
+    - RDKIT_VERSION={{ rdkit_version }}
+  number: 0
+  missing_dso_whitelist:  # [win]
+    - $RPATH/cuik_molmaker_core.dll  # [win]
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - cmake  # using CMake
```

Comment:
You're using CMake, but not using `${CMAKE_ARGS}` in the build scripts or calling CMake directly? You may have trouble with our cross-compiling environments if you don't use `${CMAKE_ARGS}`.

https://conda-forge.org/docs/maintainer/knowledge_base/#using-cmake

Reply:
You'll just have to try out the cross-compilation with what you have after your feedstock is created. Then, if it doesn't work, you'll have to read the conda-forge docs and figure out how to use the environment variables that are referenced in `${CMAKE_ARGS}` in order to setup the environment correctly.

Since it looks like you've built a custom build_ext, I can't really provide more advice on this topic.